release-notes.hbs.md

```diff
@@ -367,6 +367,12 @@ This release has the following security fixes, listed by component and area.
 
 The following issues, listed by component and area, are resolved in this release.
 
+#### <a id='1-6-0-scst-store-ri'></a> Supply Chain Security Tools - Store
+- Implemented basic logging in the AMR database.
```

Comment:
You need to add a new line between a heading and the first item of a list. The docs toolchain will not render it nicely otherwise.